lib/guardian/topic_guardian.rb

```diff
@@ -308,7 +308,7 @@ def filter_allowed_categories(records)
 
   def can_edit_featured_link?(category_id)
     return false unless SiteSetting.topic_featured_link_enabled
-    return false unless @user.trust_level >= TrustLevel.levels[:basic]
+    return false if @user.trust_level < TrustLevel.levels[:basic]
```

Comment:
There is nothing less than basic, so this could be `return false if @user.trust_level ==  TrustLevel[0]`

Reply:
For some reason `TrustLevel.levels[:new_user]` returns nil instead of 0

Reply:
That’s because it’s `:newuser` without any `_` :sweat_smile:

Reply:
@Flink That makes sense! Though it really should be `new_user`...this was added almost 10 years ago by Jeff :laughing: 
https://github.com/discourse/discourse/commit/b64a4100fafc71be8cf402467f80b6c0c672fb54#diff-529a661ec0d1f5476576e24cf6a33f445c698b3d982f9c557e9ab549d28418de